controllers/albumController.js

```diff
@@ -0,0 +1,31 @@
+import Player from '../services/Player'
+
+export const getAlbums = async (req, res) => {
+  const albums = await Player.getAlbums()
+
+  res.send(albums)
+}
+
+export const getAlbumById = async (req, res) => {
+  const { id } = req.query
+  const album = await Player.getAlbumById(id)
+
+  if (!album) {
+    return res.status(404).send('album not found')
+  }
+
+  const artist = await Player.getArtistByAlbum(album.artist_id)
+  const songs = await Player.getSongsByAlbum(id)
+  const songsIds = songs.map(s => s.id)
+  const { total } = await Player.getTotalNumberOfSongs(id)
```

Comment:
You already have songs on an album in this call `Player.getSongsByAlbum(id)`
Why do we need another DB call to get the total number of songs (`Player.getTotalNumberOfSongs(id)`)?
We can get the total with `songs.length`, right?